docs/platform/release-notes/tizen-6-0-m1.md

```diff
@@ -0,0 +1,692 @@
+- curl
+  - Version 7.62 has been upgraded to version 7.68.
+- libsoup
+  - Version 2.46.0 has been upgraded to version 2.62.2.
+- libwebsockets
+  - Version 2.3.0 has been upgraded to version 3.2.0.
+
+### Fixes
+
+- Bug in user awareness framework has been fixed.
+- Bug in bluetooth framework has been fixed.
+
+
+## Security
+
+### New and changed features
+
+- Askuser
+  - New application for user consent has been added.
+    - It was provided by a daemon, however, now it is being provided by an application.
+- Security manager
+  - The way to control access of shared/data has been changed.
+    - It is controlled in an application’s mount namespace.
+  - New privilege control mechanism has been added.
+    - Privileges can be mapped to predefined Smack label and rules.
+    - Applications that have these privileges have proper Smack rule set when it is launched.
+- Privileges
+  - New privilege has been added.
+    - notification.admin
+
+
+## Service framework
+
+### New and changed features
+
+- Battery-Monitor framework
+  - Support for fetching battery information for custom period has been added.
+  - Battery monitor API set has been changed to return usage in mAh instead of percentage.
+  - Support for C# API set for battery monitor has been added. 
+  - Support for estimating battery information for external tools has been added.
+    - Support for estimating approximate power usage per application ID has been added.
+    - Support for estimating power usage per system resource has been added.
+    - Per-app statistics of state and events has been added. 
+  - Following API set has been changed to provide custom time period support and return usage in mAh.
+    - API for fetching information for an application for a particular resource over a certain duration of time has been updated.
+    - API for fetching total battery usage information of an application, combining all the resources over certain duration of time has been updated.
+    - API for fetching battery usage values for all the resources separately used by an application ID for a certain duration has been updated.
+    - API for fetching battery usage values for a particular resource over certain duration of time has been updated.
+- Contact Framework
+  - Support for getting count for contact records has been added for better performance.
+    - API for fetching count of searched records has been added.
+    - API for fetching count of searched record with range has been added.
+    - API set for fetching count for searched record with query has been added.
+
+### Fixes
+
+- Context Framework
+  - Memory stacking issues have been resolved.
+- Contact Framework
+  - DB integrity checks have been added.
+  - Handling of XCUSTOMCHARSET has been added.
+
+
+## Web framework
+
+### New and changed features
+
+- Web Engine
+  - Open source chromium 76 version base has been applied. (below feature  and API is provided by new web engine)

```

Comment:
@nke94,

Thank you for the clarification.

Suggest rephrasing to:

- Open source chromium 76 version base has been applied. The following features and APIs are provided by the new web engine:

Reply:
Will accept but use "API set" instead of "APIs" for consistency.
Open source chromium 76 version base has been applied. The following features and API set are provided by the new web engine: